h/static/scripts/test/frame-sync-test.js

```diff
@@ -52,8 +53,14 @@ describe('FrameSync', function () {
   });
 
   beforeEach(function () {
-    fakeAnnotationUI = fakeStore({annotations: []});
-    fakeAnnotationUI.updateAnchorStatus = sinon.stub();
+    fakeAnnotationUI = fakeStore({annotations: []}, {
```

Comment:
Hmm. Question. Is there any reason we couldn't use the real store here? Surely the whole point of a redux store is that it has no side-effects of its own, so we could just use the real thing and stub out specific methods if we wanted to ensure they were called correctly?